src/main/java/com/launchacademy/reviews/seeders/BerrySeeder.java

```diff
@@ -0,0 +1,39 @@
+package com.launchacademy.reviews.seeders;
+
+import com.launchacademy.reviews.models.Berry;
+import com.launchacademy.reviews.services.BerryService;
+import java.util.List;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component
+public class BerrySeeder {
+
+  private BerryService berryService;
+
+  @Autowired
+  public BerrySeeder(BerryService berryService) {
+    this.berryService = berryService;
+  }
+
+  public void seed() {
+    if (((List<Berry>)berryService.findAll()).size() == 0) {
```

Comment:
This type casting looks redundant since the `findAll()` method is currently being cast to a List already.